<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Author</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Year</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Title</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Journal</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Volume</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Issue</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Pages</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="3">
       <widget class="QLineEdit" name="txtSearch"/>
      </item>
      <item row="11" column="0" colspan="2">
       <widget class="QPushButton" name="btnFind">
        <property name="text">
         <string>Find</string>
        </property>
       </widget>
      </item>
      <item row="11" column="2">
       <widget class="QPushButton" name="btnReset">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="3">
       <widget class="QPushButton" name="btnAdd">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="3">
       <widget class="QPushButton" name="btnRemove">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="txtAuthor"/>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spnYear"/>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="spnVolume"/>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="spnIssue"/>
      </item>
      <item row="6" column="1" colspan="2">
       <widget class="QLineEdit" name="txtPages"/>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="txtJournal"/>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="txtTitle"/>
      </item>
      <item row="10" column="0" colspan="3">
       <widget class="QComboBox" name="comboBox"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
